docs/dev/contribute.rst

```diff
@@ -71,35 +71,31 @@ for more details.
         """
         # Implementation of the function
 
-2. Build the libraries
-----------------------
+2. Build the documentation
+--------------------------
 
-.. todo::
-    Check how to automate that, running generate images and generate libraries 
-    before pull request?
-
-If the function created returns a Device object, it should be added to the
-library. You could manually upload the quickplot of this function by saving it
-to ``docs>user>images>module-containing-your-function`` under
-``your-function.png`` name. However, you can also follow these steps to
-auto-generate the qnngds library:
+After modifying, adding or deleting any function of the package, you will
+obviously want to update the package's `documentation
+<https://qnngds.readthedocs.io/en/latest/>`_. To do so, open a terminal and
+execute:
 
-#. Open a terminal in ``/path-to-qnngds/docs/user/images`` and run
-   ``plot_images.py`` file. This will automatically create png images of
-   returned devices of every function in the qnngds package.
+.. code-block:: python
```

Comment:
This doesn't seem to render correctly. Also should be bash syntax highlighting, not python I think, but that doesn't really matter much.

Reply:
well spotted thanks Reed